common/main/ar_LY.xml

```diff
@@ -46,6 +46,23 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 							<dayPeriod type="night1">↑↑↑</dayPeriod>
 							<dayPeriod type="night2">ل</dayPeriod>
 						</dayPeriodWidth>
+						<dayPeriodWidth type="narrow">
+							<dayPeriod type="night2">ل</dayPeriod>
+						</dayPeriodWidth>
```

Comment:
@macchiati as we discussed yesterday, one of the surprising things in this PR are cases like this, where a path is added that wasn't present

-fV is expected to replace ^^^ with specific value in some cases, but not (that I was aware of) to add a path that wasn't present

The code responsible is here in CLDRModify:

```
                   public void handleEnd() {
                        if (skip || isL1) {
                            return;
                        }
                        // Handle all the null cases that are in the L1 value.
                        onlyValues = null;
                        message = "fix null lateral";

                        List<String> parentChain = LocaleIDParser.getParentChain(getLocaleID());
                        String localeL1 =
                                parentChain.get(parentChain.size() - 2); // get last before root
                        CLDRFile fileL1 = factory.make(localeL1, false); // only unresolved paths
                        for (String path : fileL1) {
                            if (!pathsHandled.contains(path)) {
                                handlePath(path);
```

The message "fix null lateral" seems to imply an intention to add a path that was missing (value = null), but I don't understand the rationale

Reply:
Interestingly, this is one path for which -fV does "hardening" and -fZ doesn't. This entire locale has no changes in the draft PR for -fZ https://github.com/unicode-org/cldr/pull/3287

Reply:
BTW there were no votes in ar_LY in versions 42, 43, or 44

Reply:
Status = missing (per Survey Tool) -- however, dashboard says "provisional"; currently no "missing" items per dashboard for this locale.

The path from which it inherits has status = approved.

Questions:

(1) Why does dashboard say "provisional", when the table row says "missing"?
(2) When it gets hardened, what status should it get -- missing/provision/approved?

Reply:
The answer to question (1) is that the "✘" symbol represents any one of these three `CLDRFile.DraftStatus` enum values: `missing`, `provisional`, `unconfirmed`. Also, the Dashboard categories are based on a different enum, `NotificationCategory`; the label "provisional" in the Dashboard actually stands for `NotificationCategory.notApproved`.